webapp/backend/events.py

```diff
@@ -142,3 +144,23 @@ def delete_lambda_instance_application_connection(lambda_instance_uuid, applicat
     application = Application.objects.get(uuid=application_uuid)
     LambdaInstanceApplicationConnection.objects.get(lambda_instance=lambda_instance,
                                                     application=application).delete()
+
+
+@shared_task
+def start_stop_application(lambda_instance_uuid, application_uuid, action, app_type):
+    lambda_instance = LambdaInstance.objects.get(uuid=lambda_instance_uuid)
+    application = Application.objects.get(uuid=application_uuid)
+    instanceapplication = LambdaInstanceApplicationConnection.objects.\
+        get(lambda_instance=lambda_instance, application=application)
+    if action == 'start':
```

Comment:
When changing an object from the database, do `instanceapplication.save()` after all the changes are done, to save it back to the database.